api/app/hourlies.py

```diff
@@ -57,15 +57,30 @@ async def fetch_hourly_readings_from_db(
     return result
 
 
+def _get_time_interval(time_of_interest: datetime):
+    """ Returns the start and end datetimes for hourly readings based on given time of interest """
+    # by default, we want the past 5 days, and if available the next 10 days.
+    start_time_stamp = time_of_interest - timedelta(days=5)
+    # the UI is interested in hourly reading before and after the time of interest.
+    end_time_stamp = time_of_interest + timedelta(days=10)
+
+    return start_time_stamp, end_time_stamp
+
+
 async def get_hourly_readings(
         station_codes: List[int],
         time_of_interest: datetime) -> List[WeatherStationHourlyReadings]:
     """ Get the hourly readings for the list of station codes provided.
     Reading 5 days before, and 10 days after the time of interest are returned.
     Depending on configuration, will read from WF1 or from local database.
     """
+    start_time_stamp, end_time_stamp = _get_time_interval(time_of_interest)
 
     if wildfire_one.use_wfwx():
-        return await wildfire_one.get_hourly_readings(station_codes, time_of_interest)
+        # Limit the number of concurrent connections.
```

Comment:
I guess we're reaching a point where we can just drop this flag (not this PR! - but keep in mind).